user_templates/printer.cfg

```diff
@@ -106,6 +106,9 @@
 
 ## Beacon probe also used as virtual Z endstop. Do not forget to install the plugin and add the [beacon] section to make it work!
 # [include config/hardware/probes/beacon_virtual.cfg]
+
+## Cartographer probe also used as virtual Z endstop. Do not forget to install the plugin and add the [cartographer] section to make it work!
```

Comment:
i would precise it here (i know it is not done in beacon), and write to add a cartographer section to the overrides.